mysql/resource_grant_test.go

```diff
@@ -44,38 +44,6 @@ func TestAccGrant(t *testing.T) {
 	})
 }
 
-func TestAccGrantWithGrantOption(t *testing.T) {
-	dbName := fmt.Sprintf("tf-test-%d", rand.Intn(100))
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccGrantCheckDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccGrantConfigBasic(dbName),
-				Check: resource.ComposeTestCheckFunc(
-					testAccPrivilege("mysql_grant.test", "SELECT", true, false),
-					resource.TestCheckResourceAttr("mysql_grant.test", "grant", "false"),
-				),
-			},
-			{
-				Config: testAccGrantConfigBasicWithGrant(dbName),
-				Check: resource.ComposeTestCheckFunc(
-					testAccPrivilege("mysql_grant.test", "SELECT", true, true),
-					resource.TestCheckResourceAttr("mysql_grant.test", "grant", "true"),
-				),
-			},
-			{
-				Config: testAccGrantConfigBasic(dbName),
-				Check: resource.ComposeTestCheckFunc(
-					testAccPrivilege("mysql_grant.test", "SELECT", true, false),
-					resource.TestCheckResourceAttr("mysql_grant.test", "grant", "false"),
-				),
-			},
-		},
-	})
-}
-
```

Comment:
I was able to generalize these tests into `TestAccGrantComplex` by adding an additional parameter.

Reply:
Thanks, looks perfect!